pkgs/top-level/all-packages.nix

```diff
@@ -34933,7 +34933,12 @@ with pkgs;
     inherit (darwin) sigtool;
   };
 
-  qemu-utils = callPackage ../applications/virtualization/qemu/utils.nix { };
+  qemu-utils = callPackage ../applications/virtualization/qemu/utils.nix {
```

Comment:
Why do we still need utils.nix?  How much space are we saving?

Reply:
Some 35 MB on top of the 7 MB we had. It appears to be more useful and correct this way though.